tools/castle-editor/code/framedesign.pas

```diff
@@ -1857,6 +1863,29 @@ procedure TDesignFrame.ControlsTreeDragOver(Sender, Source: TObject; X,
   ControlsTree.Invalidate; // force custom-drawn look redraw
 end;
 
+procedure TDesignFrame.ControlsTreeEditing(Sender: TObject; Node: TTreeNode;
+  var AllowEdit: Boolean);
+begin
+  { this event is fired when calling TCustomListView.CanEdit which itself is called in TCustomListView.ShowEditor
+    therefore this event preceeds initializing and showing of the editor }
+  //here we have to "restore" the pure name of the component (without class name) before starting edit
```

Comment:
Minor: 
- Make a space between `//` and text, so `//here` -> `// here`.
- It is best to merge the 2 comments. I understand your style, that `{ }` is a comment about the method, while `// xxx` is a comment about the subsequent line -- but it's not a standard we follow in CGE.
- Start first comment with upper letter.

I am nitpicking today :) Sorry! But I would prefer to write it like this:

```
  { This event is fired when calling TCustomListView.CanEdit which itself is called in TCustomListView.ShowEditor
    therefore this event preceeds initializing and showing of the editor.

    Here we have to "restore" the pure name of the component (without class name) before starting edit. }
```

Reply:
Fixed in e7b77931ce5cd2af43489866d6a488af2ee0b311